src/web/components/Sidebar/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import computeStyle from './sidebar.style';
+
+const Sidebar = (props) => {
```

Comment:
On such a (simple) component, you can move the style computation on here 😄

Reply:
You can also destructure inside the function definition instead of below

Reply:
Done :smile: